wagtail/admin/templates/wagtailadmin/shared/headers/slim_header.html

```diff
@@ -114,7 +114,13 @@ <h1 class="w-sr-only">
                 {% endif %}
             {% endblock %}
 
-            <div class="w-w-full sm:w-w-min w-flex sm:w-flex-nowrap sm:w-flex-row w-items-center w-p-0 sm:w-py-0 sm:w-pr-4 sm:w-justify-end">
+            <div class="w-w-full w-overflow-scroll sm:w-w-min w-flex sm:w-flex-nowrap sm:w-flex-row w-items-center w-p-0 sm:w-py-0 sm:w-pr-4 sm:w-justify-end">
```

Comment:
`overflow: scroll` means scrollbar tracks are visible no matter what, so this is probably not what we want.